control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -862,6 +863,12 @@ func (r *HostedControlPlaneReconciler) reconcile(ctx context.Context, hostedCont
 		return fmt.Errorf("failed to reconcile csi driver: %w", err)
 	}
 
+	// Reconcile CSI snapshot controller operator
+	r.Log.Info("Reconciling CSI snapshot controller operator")
+	if err := r.reconcileCSISnapshotControllerOperator(ctx, hostedControlPlane, releaseImage, createOrUpdate); err != nil {
+		return fmt.Errorf("failed to ensure control plane: %w", err)
```

Comment:
this error message seems wrong.

Reply:
Filed https://github.com/openshift/hypershift/pull/1916